.github/SECURITY.md

```diff
@@ -0,0 +1,32 @@
+# Responsible Security Disclosure README standard
+
+## Our security policy and Your responsibility
+
+- **POLICY**:
+
+*Our security policy is to avoid leaving the ecosystem worse than we found it. Meaning we are not planning to introduce vulnerabilities into the ecosystem.*
+
+The "ITGlueAPI" team and community take all security bugs in "ITGlueAPI" seriously. Thank you for improving the security of "ITGlueAPI". We appreciate your efforts and responsible disclosure and will make every effort to acknowledge your contributions.
+
+Report security bugs by emailing the lead maintainer at [placeholder@example.com](placeholder@example.com) and include the word "SECURITY" in the subject line..
+
+The lead maintainer will acknowledge your email within a week, and will send a more detailed response 48 hours after that indicating the next steps in handling your report. After the initial reply to your report, the security team will endeavor to keep you informed of the progress towards a fix and full announcement, and may ask for additional information or guidance.
```

Comment:
This is good intent but not sure these timelines can be fulfilled, maybe wording that suggests best effort instead of time boxed. Welcome to other thoughts

Reply:
Ya simple boilerplate wording, how about this?

"_As mentioned above we are not planning to introduce vulnerabilities into the ecosystem and being a small team we will try to acknowledge/address security disclosures on a best effort basis. After the initial reply to your report, the security team will endeavor to keep you informed of the progress toward a fix and full announcement and may ask for additional information or guidance._"